libs/partners/mongodb/tests/integration_tests/test_vectorstores.py

```diff
@@ -30,21 +35,64 @@ def _insert_texts(self, texts: List[str], metadatas: List[Dict[str, Any]]) -> Li
         ids = super()._insert_texts(texts, metadatas)
         timeout = TIMEOUT
         while len(ids) != self.similarity_search("sandwich") and timeout >= 0:
-            sleep(INTERVAL)
+            sleep(INTERVAL * 3)
             timeout -= INTERVAL
         return ids
 
+    def create_index(
+        self,
+        dimensions: int,
+        filters: Optional[List[Dict[str, str]]] = None,
+        update: bool = False,
+    ) -> None:
+        result = super().create_index(
+            dimensions=dimensions, filters=filters, update=update
+        )
+        timeout = TIMEOUT * 3
```

Comment:
Why `TIMEOUT * 3` and `INTERVAL * 3`?

Reply:
a vestige of speed-up attempts during testing.